runtime/transformation-dom-particle.js

```diff
@@ -21,6 +21,9 @@ export class TransformationDomParticle extends DomParticle {
   getTemplate(slotName) {
     return this._state.template;
   }
+  getTemplateName(slotName) {
+    return this._state.templateName;
```

Comment:
We never need to include the slot name for the base TransformationDomParticle? May be worth a comment to explain.